### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by cvanelteren** [doomemacs/doomemacs#6792]
### What did you expect to happen?

Running install command should install doom.

```
git clone --depth 1 https://github.com/doomemacs/doomemacs ~/.emacs.d
~/.emacs.d/bin/doom install

```

### What actually happened?

```
> Synchronizing "default" profile...
  > Regenerating envvars file
    ✓ Generated ~/.emacs.d/.local/env
  x There was an unexpected runtime error
    Message: Wrong type argument
    Details: (stringp nil)
    Backtrace:
      (load nil noerror nomessage nosuffix)
      (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-p file) (progn (insert-...
      (progn (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-p file) (progn (...
      (unwind-protect (progn (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-...
      (save-current-buffer (set-buffer temp-buffer) (unwind-protect (progn (if (not noeval) (load file n...
      (let ((temp-buffer (generate-new-buffer " *temp*" t))) (save-current-buffer (set-buffer temp-buffe...
      (condition-case e (let ((temp-buffer (generate-new-buffer " *temp*" t))) (save-current-buffer (set...
      (doom--read-packages nil nil noerror)
      (while (consp --cl-var--) (setq mod (car --cl-var--) cat (car-safe (prog1 mod (setq mod (cdr mod))...
      (let* ((--cl-var-- (doom-module-list)) (mod nil) (cat nil) (path nil) (doom--current-module nil) (...
      (if all-p (mapc #'doom--read-packages (doom-files-in doom-modules-dir :depth 2 :match "/packages\\...
      (let ((private-packages (doom-path doom-user-dir packages-file))) (if all-p (mapc #'doom--read-pac...
    ! Wrote extended backtrace to ~/.emacs.d/.local/logs/cli.doom.220916185944.227235.error
  ! Script was abruptly aborted, leaving Doom in an incomplete state!
  - Run 'doom sync' to repair it.
  
 ```

### Describe your attempts to resolve the issue

Running `./doom sync` does not solve the issue but reveals:

```
! Failed to locate a ':tools +python' module
> Synchronizing "default" profile...
  > Regenerating envvars file
    ✓ Generated ~/.emacs.d/.local/env
  x There was an unexpected runtime error
    Message: Wrong type argument
    Details: (stringp nil)
    Backtrace:
      (load nil noerror nomessage nosuffix)
      (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-p file) (progn (insert-...
      (progn (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-p file) (progn (...
      (unwind-protect (progn (if (not noeval) (load file noerror 'nomessage 'nosuffix) (if (file-exists-...
      (save-current-buffer (set-buffer temp-buffer) (unwind-protect (progn (if (not noeval) (load file n...
      (let ((temp-buffer (generate-new-buffer " *temp*" t))) (save-current-buffer (set-buffer temp-buffe...
      (condition-case e (let ((temp-buffer (generate-new-buffer " *temp*" t))) (save-current-buffer (set...
      (doom--read-packages nil nil noerror)
      (while (consp --cl-var--) (setq mod (car --cl-var--) cat (car-safe (prog1 mod (setq mod (cdr mod))...
      (let* ((--cl-var-- (doom-module-list)) (mod nil) (cat nil) (path nil) (doom--current-module nil) (...
      (if all-p (mapc #'doom--read-packages (doom-files-in doom-modules-dir :depth 2 :match "/packages\\...
      (let ((private-packages (doom-path doom-user-dir packages-file))) (if all-p (mapc #'doom--read-pac...
    ! Wrote extended backtrace to ~/.emacs.d/.local/logs/cli.doom.220916190338.227566.error
  ! Script was abruptly aborted, leaving Doom in an incomplete state!
  - Run 'doom sync' to repair it.
```

### Steps to reproduce

```
git clone --depth 1 https://github.com/doomemacs/doomemacs
cd ~/.emacs.d
./doom install
```

### System Information

https://pastebin.com/SsFLrb4C

**Comment by hlissner** [comment]
Duplicate of #6777.

**Comment by cvanelteren** [comment]
There is no fix listed in #6777, I am on commit 811c18ca680d31b411ce395e0d28ab97816f177a, any pointers @hlissner ?

**Comment by cvanelteren** [comment]
For posterity. There was for some reason a parenthesis missing in my init.el. All is good now.
